src/supportsProtoAssignment.js

```diff
@@ -0,0 +1,3 @@
+export default function supportsProtoAssignment (entity) {
+  return !!entity && (entity.__proto__ instanceof Object)
+};
```

Comment:
I really debated just making this a global check on the object primitive, but this is super easy to test with `Object.create(null)`.